server/services/store/sqlstore/board.go

```diff
@@ -528,3 +580,30 @@ func (s *SQLStore) searchBoardsForUserAndTeam(db sq.BaseRunner, term, userID, te
 
 	return s.boardsFromRows(rows)
 }
+
+func (s *SQLStore) getBoardMemberHistory(db sq.BaseRunner, boardID, userID string, limit uint64) ([]*model.BoardMemberHistoryEntry, error) {
```

Comment:
This function and the `boardMemberHistoryEntriesFromRows` are just to be able to test it properly as part of the store, also, we are going to need something like that for the compliance export sooner than later, so I'm doing it anyway.